Bio/SeqIO/SffIO.py

```diff
@@ -1030,112 +883,228 @@ def iterate(self, handle):
         # clip_adapter_left      H
         # clip_adapter_right     H
         # [rest of read header depends on the name length etc]
-        read_header_fmt = ">2HI4H"
-        read_header_size = struct.calcsize(read_header_fmt)
-        read_flow_fmt = ">%iH" % number_of_flows_per_read
-        read_flow_size = struct.calcsize(read_flow_fmt)
-        assert 1 == struct.calcsize(">B")
-        assert 1 == struct.calcsize(">s")
-        assert 1 == struct.calcsize(">c")
-        assert read_header_size % 8 == 0  # Important for padding calc later!
-        # The spec allows for the index block to be before or even in the middle
-        # of the reads. We can check that if we keep track of our position
-        # in the file...
```

Comment:
I think this comment is worth preserving (now in the ``__next__``function where the code to skip over the index block is not explained).

Reply:
OK